docs/guide/repository.md

```diff
@@ -752,7 +620,6 @@ $cacheService->clear();
 // To clear a specific cache item (check source code in eZ\Publish\Core\Persistence\Cache\*Handlers for further info)
 $cacheService->clear('content', 'info', $contentId);
 
-// Stash cache is hierarchical, so you can clear all content/info cache like so:
 $cacheService->clear('content', 'info');
```

Comment:
all the clear examples needs to be rewritten

Reply:
Could you point me to where I can find the up-to-date way to do this?

Reply:
ha :)
It needs to be written.

In this case it's _(getting the pool is explained above in the doc)_:
```php  
// To clear all cache
$pool->clear();
  
// To clear a specific cache item (check source code in eZ\Publish\Core\Persistence\Cache\*)
$pool-> deleteItems(["ez-content-info-$contentId"]);
  
// Symfony cache is tag based, so you can clear all cache related to a content like so:
$pool-> invalidateTags(["content-$contentId"]);
```